CHANGELOG.md

```diff
@@ -46,6 +46,61 @@ Those changes in added, changed or breaking changes, should include usage exampl
         .unwrap();
   ```
 
+*  Alternate impl for OperatorsInfo for retrieving pub keys and socket for the operator from middleware directly in [#414](https://github.com/Layr-Labs/eigensdk-rs/pull/414)
+
+Old Implementation which uses indexing :
+
+  ```rust 
+    use eigen_services_operatorsinfo::{operatorsinfo_inmemory::OperatorInfoServiceInMemory};
+    let operators_info = OperatorInfoServiceInMemory::new(
+        get_test_logger(),
+        avs_registry_reader.clone(),
+        ws_endpoint,
+    )
+    .await
+    .unwrap()
+    .0;
+
+    let cancellation_token = CancellationToken::new();
+    let operators_info_clone = operators_info.clone();
+    let token_clone = cancellation_token.clone();
+    task::spawn(async move { operators_info_clone.start_service(&token_clone, start_block, end_block).await });
+    // Sleep to wait for the operator info service to start
+    sleep(Duration::from_secs(1)).await;
+
+    let avs_registry_service =
+        AvsRegistryServiceChainCaller::new(avs_registry_reader.clone(), operators_info);
+
+    // Send the shutdown signal to the OperatorInfoServiceInMemory
+    cancellation_token.cancel();
```

Comment:
I think including this part might be confusing. This should be run only after we no longer need both services.

Reply:
👍